From 7fc56e7d873d0dcbcdf59fde6ae17e632898207c Mon Sep 17 00:00:00 2001
From: scx <scx.mail@gmail.com>
Date: Wed, 29 Jan 2020 20:33:04 +0100
Subject: [PATCH] Fix PDF import: execute Ghostscript in proper way

Some changes are required to execute commands in Python3 properly.

Fixes GNOME/ocrfeeder#19
---
 src/ocrfeeder/util/lib.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/ocrfeeder/util/lib.py b/src/ocrfeeder/util/lib.py
index ff9b314..7766230 100644
--- a/src/ocrfeeder/util/lib.py
+++ b/src/ocrfeeder/util/lib.py
@@ -54,7 +54,7 @@ def convertPdfToImages(pdf_file, temp_dir = '/tmp'):
                'file_name': file_name,
                'pdf_file': pdf_file,
                'resolution': resolution}
-    os.popen(command)
+    os.popen(command).read()
     return dir_name
 
 def getImagesFromFolder(folder):
-- 
2.24.1

